src/screen/Council/CouncilScreen.tsx

```diff
@@ -132,17 +173,197 @@ function Item({
   );
 }
 
+type CouncilVoteProps = {
+  visible: boolean;
+  setVisible: (visible: boolean) => void;
+  candidates: string[];
+  module: string | null;
+};
+
+function CouncilVote({visible, setVisible, candidates, module}: CouncilVoteProps) {
+  const [account, setAccount] = React.useState<string>();
+  const [amount, setAmount] = React.useState<string>('');
+  const [selectedCandidates, setSelectedCandidates] = React.useState<Array<string>>([]);
+
+  const {api} = useApi();
+  const startTx = useApiTx();
+  const {networkAccounts} = useAccounts();
+  const formatBalance = useFormatBalance();
+
+  const onCandidateSelect = (accountId: string, isSelected: boolean) => {
+    isSelected
+      ? setSelectedCandidates(selectedCandidates.filter((id) => id !== accountId))
+      : selectedCandidates.length < MAX_VOTES
+      ? setSelectedCandidates([...selectedCandidates, accountId])
+      : noop;
+  };
+
+  const disabled = !amount || !account || !(selectedCandidates.length > 0);
```

Comment:
```suggestion
  const disabled = !amount || !account || selectedCandidates.length === 0;
```

Reply:
What if the user has already voted with an account? Can we get from the chain if that's the case? Can he/she vote again?

Reply:
We can get for which council members user has already voted with an account and preselect them from the list but a user can vote multiple times for same council members using the same account.